packages/p2p/src/components/my-ads/my-ads-table.jsx

```diff
@@ -145,19 +145,7 @@ const MyAdsTable = () => {
         );
     }
 
-    return (
-        <Empty icon='IcCashierNoAds' title={localize('You have no ads.')}>
-            <Button
-                className='p2p-empty__button'
-                is_disabled={general_store.is_barred}
-                onClick={() => my_ads_store.onClickCreate()}
-                large
-                primary
-            >
-                {localize('Create new ad')}
-            </Button>
-        </Empty>
-    );
+    return <NoAds is_ads_page={true} />;
```

Comment:
Could you try this? Cause this usually works as well, if not, can ignore my comment

```suggestion
    return <NoAds is_ads_page/>;
```